themes/Mumble/README.md

```diff
@@ -1,31 +1,7 @@
-A **Mumble theme in Lite and Dark variants**, which is **distributed as the default skin in the Mumble client**.
+# The Mumble theme
 
-This theme is a derivative of the [MetroMumble skin](https://github.com/xPoke/MetroMumble).
+This theme is a derivative of the [MetroMumble skin](https://github.com/xPoke/MetroMumble). It comes in a Lite and a Dark variant as well with the
```

Comment:
typo missing second as on *as well as*

I think the sentence could be simplified:

It comes in a Lite and a Dark variant and variants that are specifically tuned for use on macOS.